package.json

```diff
@@ -79,16 +79,17 @@
     "xml-js": "^1.6.11"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "start:server": "node build/server.js",
+    "start": "TZ=Europe/Helsinki react-scripts start",
+    "start:server": "TZ=Europe/Helsinki node build/server.js",
     "build": "yarn build:client && yarn build:server",
     "build:client": "react-scripts build",
     "build:server": "NODE_ENV=production webpack --progress --mode production --config config/webpack.server.config.js",
     "codegen": "graphql-codegen -r dotenv/config dotenv_config_path=.env.development.local  --config codegen.yml",
-    "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
-    "test:staged": "react-scripts test --env=jest-environment-jsdom-sixteen --watchAll=false --findRelatedTests",
-    "test:coverage": "react-scripts test --coverage --env=jest-environment-jsdom-sixteen --watchAll=false",
-    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
+    "test": "TZ=Europe/Helsinki react-scripts test --env=jest-environment-jsdom-sixteen",
+    "ci": "CI=true yarn test --verbose --runInBand --coverage",
```

Comment:
is runInBand more performant in github action?

Reply:
Not sure, this was just used in venepaikka script (for ci only) and counted on that.

Reply:
no difference. Removed runInBand